tools/testplan_vs_workspace.py

```diff
@@ -0,0 +1,131 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2021, Codecoup.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+"""Script for printing diff between testplan.xlsx and active test cases in workspace
+
+Usage:
+$ python3 testplan_vs_workspace.py path/to/workspace path/to/test_plan.xlsx <project_name>
+"""
+import importlib
+import sys
+import os
+import pandas as pd
+import win32com
+import wmi
+

```

Comment:
Add this to be able to run from project folder from command line.
```suggestion
sys.path.insert(
    0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```